configs/getConfig.js

```diff
@@ -0,0 +1,20 @@
+const _ = require("lodash");
```

Comment:
I think it's a little confusing for code and configs to share the same directory. I'm still getting my head around this new structure so I don't know what to propose yet, but ideally the top-level `config` directory would just contain json files.